packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```
@@ -23,19 +23,23 @@
 %global gem_name smart_proxy_salt
 %global plugin_name salt
 
-%global foreman_proxy_min_version 1.25
+%global foreman_proxy_min_version 2.5.2
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
 %global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
 %global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
 
 %global salt_config_dir %{_root_sysconfdir}/salt
+%global salt_config_runners_dir /var/lib/foreman-proxy/salt/runners
+%global salt_config_reactors_dir /var/lib/foreman-proxy/salt/reactors
+%global salt_config_grains_dir /var/lib/foreman-proxy/salt/grains
```

Please reuse %{foreman_proxy_statedir}
```suggestion
%global salt_config_runners_dir %{foreman_proxy_statedir}/salt/runners
%global salt_config_reactors_dir %{foreman_proxy_statedir}/salt/reactors
%global salt_config_grains_dir %{foreman_proxy_statedir}/salt/grains
```